.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,51 +1,41 @@
 
 <!--
-^ Please summarise the changes you have done and explain why they are necessary here ^
-
-For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
-For new packages please briefly describe the package or provide a link to its homepage.
+Please summarise your changes and explain why they are necessary here.
 -->
 
 ## Things done
 
-<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->
+<!-- Please check what applies. Not all of these are hard requirements. -->
 
-- Built on platform(s)
+- Built on platform:
   - [ ] x86_64-linux
   - [ ] aarch64-linux
   - [ ] x86_64-darwin
   - [ ] aarch64-darwin
-- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
-  - [ ] `sandbox = relaxed`
-  - [ ] `sandbox = true`
```

Comment:
Maybe we can include this within the platforms list above?

E.g. something like

```md
- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] x86_64-darwin (sandbox = off)
  - [ ] x86_64-darwin (sandbox = relaxed)
  - [ ] aarch64-darwin (sandbox = off)
  - [ ] aarch64-darwin (sandbox = relaxed)
```

Not ideal though...

I'm also happy with simply removing the sandbox section, as most simply ignore it anyway.